apm-agent-plugins/apm-spring-webmvc/apm-spring-webmvc-spring5/src/main/java/co/elastic/apm/agent/springwebmvc/AbstractSpringExceptionHandlerInstrumentation.java

```diff
@@ -63,7 +67,18 @@ public static <HttpServletRequest> void captureRequestError(ServletRequestAdapte
                                                                     @Nullable HttpServletRequest request,
                                                                     @Nullable Exception e) {
             if (request != null && e != null) {
-                adapter.setAttribute(request, "co.elastic.apm.exception", e);
+                Tracer tracer = GlobalTracer.get();
+                boolean notUseAttribs = tracer.getConfig(CoreConfiguration.class).isNotUseServletAttributesForExceptionPropagation();
+                Throwable maybeRedacted = tracer.redactExceptionIfRequired(e);
+
+                if (notUseAttribs) {
```

Comment:
[minor] here switching to a "positive" wording for the config option would allow to use the simplest case first.